src/frontend/components/movie-details/MovieDetails.js

```diff
@@ -1,59 +1,78 @@
 import React, {Component} from 'react';
 import {Redirect} from 'react-router-dom';
-import Movies from '../tv-shows/gallery/GalleryGet';
+//import Movies from '../tv-shows/gallery/GalleryGet';
 import './MovieDetails.css';
+import fetchMovie from '../tv-shows/gallery/galleryObject';
 
 class MovieDetails extends Component{
     constructor(){
         super();
         this.state = {
+            objLoaded: false,
+            image: 'empty',
             message: 'Hello, this will be the '
                 +'details page for',
-            movieObject: {}
+            movieObject: {
+                
+            },
         };
     }
 
     componentWillMount(){
-        //frontend
-        let movieObject = this.findMovieObject(Movies, this.props.match.params.movie);
-        this.setState({
-            movieObject: movieObject
-        });
+       //do nothing
+    }
+
+    componentDidMount(){
+        console.log('It is mounted');
+        fetchMovie.getMovie();
+        console.log(fetchMovie.returnedObj);
+        console.log('Here are the movies');
+
         //backend
-        // let movieObj  = fetch('/rest/show')
+        // fetch('/rest/show')
         //     .then((res)=> {
-
+        //         return res.json();
         //     })
-        //     .then((res)=>{
-
+        //     .then((resJson)=>{
+        //         let movieObj = this.findMovieObject(resJson, this.props.match.params.movie);
+        //         this.setState({
+        //             objLoaded: true,
+        //             backendObject: movieObj,
+        //         });
         //     });
     }
-
-    componentDidMount(){
-        //do nothing
-    }
-
     render(){
```

Comment:
Please remove, at least, the console.logs here. You should use devTools to debug these things :)